app/models/manageiq/providers/redhat/infra_manager/vm.rb

```diff
@@ -6,40 +6,39 @@ class ManageIQ::Providers::Redhat::InfraManager::Vm < ManageIQ::Providers::Ovirt
 
   supports :migrate do
     if blank? || orphaned? || archived?
-      unsupported_reason_add(:migrate, "Migrate operation in not supported.")
-    elsif !ext_management_system.supports?(:migrate)
-      unsupported_reason_add(:migrate, 'RHV API version does not support migrate')
+      "Migrate operation in not supported."
+    else
+      ext_management_system.unsupported_reason(:migrate)
     end
   end
 
   supports :reconfigure_disks do
     if storage.blank?
-      unsupported_reason_add(:reconfigure_disks, _('storage is missing'))
+      _('storage is missing')
     elsif ext_management_system.blank?
-      unsupported_reason_add(:reconfigure_disks, _('The virtual machine is not associated with a provider'))
-    elsif !ext_management_system.supports?(:reconfigure_disks)
-      unsupported_reason_add(:reconfigure_disks, _('The provider does not support reconfigure disks'))
+      _('The virtual machine is not associated with a provider')
+    else
+      ext_management_system.unsupported_reason(:reconfigure_disks)
     end
   end
 
   supports_not :reset
   supports :publish do
     if blank? || orphaned? || archived?
```

Comment:
yea. not sure what `blank?` means. think it wanted to be `ems.blank?`, which happens to be handled in this case.

Reply:
Yeah not sure what blank? is here either...